File: 724-find-pivot-index/724-find-pivot-index.c
int pivotIndex(int *nums, int numsSize)
{

    int totalSum = 0;
    for (int i = 0; i < numsSize; i++)
        totalSum += nums[i];

    for (int i = 0, leftSum = 0; i < numsSize; i++)
    {
        if (i>0)
            leftSum += nums[i - 1];
        if (leftSum * 2 == totalSum - nums[i])
            return i;
    }
    return -1;
}

File: 724-find-pivot-index/724-find-pivot-index.py
class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        totalSum = 0
        for val in nums:
            totalSum+=val

        for i in range(len(nums)):
            leftSum = sum(nums[:i])
            if (leftSum * 2) == (totalSum - nums[i]):
                return i

        return -1